server/graphql/resolvers/members/mutation/deleteSkillsToMember.ts

```diff
@@ -12,10 +12,9 @@ const deleteSkillsFromMember = async (
     const { memberID, skills } = args.request;
     console.log("Mutation > DeleteSkillsFromMember > args.fields = ", args.request);
 
-    if (!memberID) throw new Error("memberID (from Discord) is required to update member");
-    if (memberID.length !== 18) throw new Error("memberID invalid");
+    if (!memberID) throw new Error("memberID (from mongoDB _id) is required to update member");
```

Comment:
We are using `memberID` to check `_id` in one query and `discordID` in one query. This will be a problem. We should still to check with either one of them, or change the key from `memberID` to `_id` or `discordID` based on usage